include/types.hpp

```diff
@@ -36,5 +36,11 @@ using DbusVariantType = std::variant<
  using MapperGetObject =
     std::vector<std::pair<std::string, std::vector<std::string>>>;
 
+using ParsedVPD = std::unordered_map<std::string,
+                                  std::unordered_map<std::string, std::string>>;
+using BinaryArray = std::vector<uint8_t>;
```

Comment:
Better to use the name BinaryVector for better readability  instead of BinaryArray